tests/e2e/modules/newsfeed_spec.js

```diff
@@ -36,6 +36,10 @@ describe("Newsfeed module", function () {
 		it("should show the newsfeed article", function () {
 			return app.client.waitUntilTextExists(".newsfeed .newsfeed-title", "QPanel", 10000);
 		});
+
+		it("should show the newsfeed description", function () {
+			return app.client.waitUntilTextExists(".newsfeed .newsfeed-desc", "Para instalar esta nueva versión", 10000);
+		});
```

Comment:
And maybe a test where the description is turned off *prettyplease* :-)

Reply:
Done. Although I'm not sure if this is the right way to test for the absence of a text

Reply:
Mhhh, maybe wait till the title exists and then check if the description is there? Might also be nice to do in the other test

Reply:
I'm not too familiar with these tests. I tried but couldn't get that to work...